### ISSUE TITLE
Make Karnofsky Value Set reference LOINC

### ISSUE THREAD

**Issue by markkramerus** [standardhealth/fsh-mcode#38]
KarnofskyPerformanceStatusVS is a copy of LOINC list LL4986-7. We did last year this because of IG Pub limitations. I wonder if it is time to change it to a reference to the LOINC answer set.  The use of LOINC answer lists in FHIR is explained here: https://www.hl7.org/fhir/loinc.html#alist

**Comment by mlterryMitre-zz** [comment]
@markkramerus - Looking this over, I'm not sure I agree with binding the code to LOINC list LL4986-7.  LOINC models Karnofsky as a panel which separates out code and interpretation whereby LL4986-7 is the interpretation.  Binding the value to valueCodeableConcept to only the interpretation does not allow storage of the actual numeric value, and it is the numeric value which is most relevant to both Karnofsky.  Same goes with ECOGPerformanceStatus, which mCODE modeled in the same way.
In short, the way we modeled Karnofsky does not fit what LOINC did.  We can change it to LL but it doesn't fix the greater issue of modeling each to accomodate both code and interpretation.

**Comment by markkramerus** [comment]
Tried to find a precedent in International Patient Summary. Didn't find one. To quote, "The functional status section contains a narrative description of capability of the patient to perform acts of daily living, including possible needs of the patient to be continuously assessed by third parties. The invalidity status may in fact influence decisions about how to administer treatments. Profiles to express disabilities and functional assessments will be specified by future versions of this guide."   http://hl7.org/fhir/uv/ips/2018May/StructureDefinition-composition-uv-ips.html#functional-status

**Comment by mlterryMitre-zz** [comment]
fsh-mcode PR# https://github.com/standardhealth/fsh-mcode/pull/83 created with the following fixes:
- valueCodeableConcept changed to valueInteger
- bound interpretation to the LOINC answer list
